api/migrations/20210812174148-create-guild.js

```diff
@@ -0,0 +1,45 @@
+"use strict";
+module.exports = {
+  up: async (queryInterface, Sequelize) => {
+    await queryInterface.createTable("guilds", {
+      id: {
+        allowNull: false,
+        type: Sequelize.STRING,
+      },
+      prefix: {
+        defaultValue: ".",
```

Comment:
What does prefix mean and why do we pick this as the default.

Reply:
The prefix is what users need to add in front of the message to indicate that it is a command. For example `.help`, `.balance` etc.
We want the prefix to be customizable, so there will be a command that will allow mods to change the prefix in their correspondent guild.